src/languages/python.js

```diff
@@ -373,6 +373,7 @@ export default function(hljs) {
       NUMBER,
       {
         // very common convention
+        scope: 'variable.language',
         begin: /\bself\b/
       },
```

Comment:
You'll need to pull this out into a `SELF_MODE` and make sure it's include both here and above around line 346 (so it's highlighting inside params too) - or was that something you omitted on purpose?

And ignore the literal `"self"` that's just telling it params can include more params inside it's self recursively to allow for nested `()` expressions...

Reply:
Ping.

Reply:
Hi @joshgoebel,

Sorry for the late response. Good thoughts. I'll close this pull request and open another once I have those changes ready.